integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java

```diff
@@ -399,120 +394,86 @@ private static void createDomainCrAndVerify(String adminSecretName,
                                               String encryptionSecretName,
                                               String miiImage) {
     final String volumeName = "weblogic-domain-storage-volume";
-    final String fluentdRootPath = "/scratch";
+    final String logHomeRootPath = "/scratch";
     // create the domain CR
+
+    FluentdSpecification fluentdSpecification = new FluentdSpecification();
+    fluentdSpecification.setImage(FLUENTD_IMAGE);
+    fluentdSpecification.setWatchIntrospectorLogs(false);
+    fluentdSpecification.setImagePullPolicy("IfNotPresent");
+    fluentdSpecification.setElasticSearchCredentials("weblogic-credentials");
+    fluentdSpecification.setVolumeMounts(Arrays.asList(new V1VolumeMount()
+            .name(volumeName)
+            .mountPath(logHomeRootPath)));
+
+    try {
+      Path filePath = Path.of(MODEL_DIR + "/" + FLUENTD_CONFIGMAP_YAML);
```

Comment:
Instead of catching an Exception you could wrap this in assertDoesNotThrow()

Reply:
fixed